clients/common/utility.cpp

```diff
@@ -71,8 +71,15 @@ rocblas_int query_device_property()
         printf("Query device success: there are %d devices \n", device_count);
     }
 
-    for(rocblas_int i = 0; i < device_count; i++)
+    for(rocblas_int i = 0;; i++)
```

Comment:
The original code looked odd -------------------------
Like this.
----------------------------------------------

The new format is:

--------------------------------------------------
Device ID 0
...
--------------------------------------------------
Device ID 1
...
--------------------------------------------------

That is the reason for moving the loop exit into the middle of the loop, after the printing of the horizontal lines.